Customizing/global/plugins/Services/UIComponent/UserInterfaceHook/GEVCockpit/classes/class.ilGEVCockpitUIHookGUI.php

```diff
@@ -199,9 +198,27 @@ protected function getSubMenuHTML($current_skin) {
 			$tpl->parseCurrentBlock();
 			$count++;
 		}
+
+		$tpl->addCss("./Customizing/global/plugins/Services/UIComponent/UserInterfaceHook/GEVCockpit/templates/jquery.bxslider.css");
+		$js_tpl = new ilTemplate("./Customizing/global/plugins/Services/UIComponent/UserInterfaceHook/GEVCockpit/templates/tpl.submenu_slider.js.html", false, false);
```

Comment:
Ist dieses Template schon in diesem Repo? Wenn nicht, bitte hinzufügen. Ich verstehe außerdem nicht, warum das ein eigenes Template ist, wir könnten das JS doch auch einfach in tpl.submenu.html hinzufügen, oder? Ich sag nicht, dass das falsch ist, interessiere mich aber für die Gründe für die Entscheidung.